e2e/utils/TestUtils.ts

```diff
@@ -27,11 +27,35 @@ function createSettingsFile(settingsDir: string) {
       fs.writeFileSync(path.join(settingsDir, fileSettingsName), settingsJson);
       console.log('Default settings file successfully created on: ', `${ settingsDir }/${ fileSettingsName }`);
     }
-  } catch (err) {
-    console.error('Error during default settings creation. Error: --> ', err);
+  } catch (ex) {
+    console.error('Error during default settings creation. Error: --> ', ex);
+    throw ex;
```

Comment:
The logging here doesn't really provide that much information (the same stack trace is already printed, as per your samples).  Same as the one in `playwrightReportAssets`.